src/octopeer-github/main/ElementID.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Created by Youri on 08/05/2016.
+ */
+/**
+ * Class to hold elementID's so they can be a semantically different data type.
+ */
+class ElementID {
```

Comment:
Please consider moving `EventID` and `ElementID` inside a folder, I think they either belong to `DatabaseAdaptable` or `ElementSelectionBehaviour` (my preference goes to the first).